packages/replay/test/unit/util/addEvent.test.ts

```diff
@@ -29,4 +34,97 @@ describe('Unit | util | addEvent', () => {
 
     expect(replay.isEnabled()).toEqual(false);
   });
+
+  it.each([
```

Comment:
These are the most relevant tests here, making sure it works the same with & without compression for both error & session mode.